rules/windows/process_creation/win_adfind_usage_fin6.yml

```diff
@@ -0,0 +1,36 @@
+title: Environment Discovery Using AdFind
+id: 72be847f-9c5c-47c6-9cfe-510f5947075e
+description: Detects the use of the AdFind tool for discovery purposes
+author: Jonhnathan Ribeiro
+references:
+    - https://github.com/center-for-threat-informed-defense/adversary_emulation_library/blob/master/fin6/Emulation_Plan/Phase1.md
+    - http://www.joeware.net/freetools/tools/adfind/
+date: 2020/09/15
+tags:
+    - attack.discovery
+    - attack.t1087.002
+    - attack.t1018
+    - attack.t1482
+    - attack.t1016
+    - attack.t1069.002
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection1:
+        CommandLine|contains: 
+          - "-f"
+    selection2:
+        CommandLine|contains:
+          - "objectcategory=person"
+          - "objectcategory=computer"
+          - "objectcategory=organizationalUnit"
+          - "objectcategory=subnet"
+          - "objectcategory=group"
+    selection3:
+        CommandLine|contains:
+          - "trustdmp"
+    condition: selection1 and selection2 or selection3
```

Comment:
Is it mean that way? 
Only selection3 is sufficient to trigger that rule. 

Or did you mean `selection1 and (selection2 or selection3)`

Reply:
Yeah, The trustdmp command doesn't need of the -f parameter.